src/components/Home.jsx

```diff
@@ -0,0 +1,585 @@
+// @flow
+import * as React from 'react';
+import { format } from 'd3';
+import {
+        Container,
+        Grid,
+        List,
+        ListItem,
+        ListItemIcon,
+        ListItemText,
+        Typography,
+        withStyles
+} from '@material-ui/core';
+import DownTrendIcon from '@material-ui/icons/ArrowDropDown';
+import UpTrendIcon from '@material-ui/icons/ArrowDropUp';
+import FlatTrendIcon from '@material-ui/icons/FiberManualRecord';
+import Control from 'gd-core/src/components/ol/Control';
+import { createEmpty as createEmptyExtent, extend as extendExtent } from 'ol/extent';
+import GeoJSON from 'ol/format/GeoJSON';
+import GroupLayer from 'ol/layer/Group';
+import ImageLayer from 'ol/layer/Image';
+import TileLayer from 'ol/layer/Tile';
+import VectorLayer from 'ol/layer/Vector';
+import ImageWMSSource from 'ol/source/ImageWMS';
+import OSM, { ATTRIBUTION as OSM_ATTRIBUTION } from 'ol/source/OSM';
+import VectorSource from 'ol/source/Vector';
+import XYZ from 'ol/source/XYZ';
+import { decode } from 'geobuf';
+import Pbf from 'pbf';
+import { Map, BaseControlPortal } from 'gd-core/src/components/ol';
+import { entries } from 'gd-core/src/utils/array';
+import { SLRSlope } from 'gd-core/src/utils/math';
+
+import type {
+        Feature as FeatureType,
+        Map as MapType,
+        MapBrowserEventType
+} from 'ol';
+import type { Layer as LayerType } from 'ol/layer';
+
+// TODO: Is it possible to make these dynamic? import data struct from config?
+import annualYieldData from '../../data/annual_yield.json';
```

Comment:
Yes, we can use urls instead and adjust the code so it fetches the data from the given URLs first and sotree them either in the component or in redux store.